modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/ConfigModelLoaderDelegate.java

```diff
@@ -31,6 +31,7 @@
 final class ConfigModelLoaderDelegate extends AbstractModelLoaderDelegate {
 
   private static final String CONFIGURATION = "Configuration";
+  private static final String CUSTOM_CONFIG_SUFFIX = "-" + DEFAULT_CONFIG_NAME;
```

Comment:
do not tie the suffix to the default config name. simply use the `-config` literal